packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -212,7 +212,9 @@ function WorklistDataGrid(props) {
   const [focusedCol, setFocusedCol] = useState(0);
   const [ariaLiveMessage, setAriaLiveMessage] = useState(null);
   const [cellAriaLiveMessage, setCellAriaLiveMessage] = useState(null);
-  const rowSelection = useRef({ isRowSelectionModeToggledByGrid: false });
+  const isRowSelectionModeToggledByGrid = useRef(false);
+  const inShiftUpDownMode = useRef(false);
+  const multiSelectRange = useRef({ start: null, end: null });
```

Comment:
We should create more meaningful variable names than "start" and "end"

Reply:
Doesn't the variable name `multiSelectRange` already add meaning to these variables?

Reply:
The multiSelectRange gives the context and meaning to the start and end. The start and end are the start and end of the range.